kobo/apps/subsequences/integrations/google/google_transcribe.py

```diff
@@ -1,6 +1,7 @@
 # coding: utf-8
 import uuid
 import posixpath
+from __future__ import annotations
```

Comment:
`from __future__ import annotations` should be the first line of the imports and should be in its own block 

```python 
from __future__ import annotations 

import uuid
imporrt posixpath 
```